Rakefile

```diff
@@ -429,6 +429,8 @@ task :ci do
     sh 'bundle exec rake spec:opentracer'
 
     if RUBY_PLATFORM != 'java'
+      # Benchmarks
```

Comment:
Looks like we don't have separate sections for these newer versions of Ruby as we do for previous versions. We probably want to expand this out, because there will be some integrations (namely Rails 6) that will/won't run on a subset of this range of Ruby versions. It's verbose yes, but it is clear and relatively easy to work with.

Reply:
Done! Also I've done the same thing for appraisals.